pkg/asset/manifests/aws/zones.go

```diff
@@ -232,6 +269,50 @@ func setSubnetsManagedVPC(in *zonesInput) error {
 			})
 		}
 	}
+
+	// Create subnets from edge zone pool with type local-zone.
+	if len(allEdgeZones) > 0 {
```

Comment:
What about we invert the check here and return early? That would decrease the indent level by 1.
```suggestion
    // no edge zones, nothing else to do
	if len(allEdgeZones) == 0 {
	    return nil
	}
```